sql/moz-fx-data-shared-prod/ads_derived/desktop_tiles_forecast_inputs_v1/schema.yaml

```diff
@@ -0,0 +1,49 @@
+fields:
+  - mode: NULLABLE
+    name: submission_month
+    type: DATE
+    description: Data aggregation month
+  - mode: NULLABLE
+    name: country
+    type: STRING
+    description: Two-letter code corresponding to country where sponsored tiles are available. Please see `static.country_codes_v1`
+  - mode: NULLABLE
+    name: user_count
+    type: INTEGER
+    description: Count of desktop users for submission_month. Calculated from Unified Metrics.
+  - mode: NULLABLE
+    name: impression_count_1and2
+    type: INTEGER
+    description: Count of tile impressions in positions 1 and 2 for submission_month. Calculated from contextual-services.
+  - mode: NULLABLE
+    name: sponsored_impressions_all
+    type: INTEGER
+    description: Count of tile impressions across all three positions for submission_month. Calculated from contextual-services.
+  - mode: NULLABLE
+    name: visit_count
+    type: INTEGER
+    description: Count of distinct newtab visits for submission_month. Calculated from latest Newtab.
+  - mode: NULLABLE
+    name: clients
+    type: INTEGER
+    description: Count of distinct newtab clients opening the Newtab page for submission_month. Calculated from Newtab.
+  - mode: NULLABLE
+    name: total_inventory_1and2
+    type: INTEGER
+    description: Count of sponsored tile inventory for positions 1 and 2 for submission_month. Calculated from latest Newtab data.
+  - mode: NULLABLE
+    name: visits_total_fill_rate_1and2
```

Comment:
This field doesn't appear in the query; did you mean `visits_total_fill_rate_1to3`?
📎

Reply:
Should be `fill_rate`, pushed a fix